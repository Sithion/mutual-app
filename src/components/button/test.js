import 'react-native';
import React from 'react';
import renderer from 'react-test-renderer';
import Button from './button';

test('render "Button" correctly', () => {
  const container = renderer.create(
    <Button backgroundColor="black" color="white" onPress={() => null} count={1} />).toJSON();
  expect(container).toMatchSnapshot();
});
