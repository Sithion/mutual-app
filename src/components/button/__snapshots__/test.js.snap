// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render "Button" correctly 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "flex": 1,
      "justifyContent": "center",
      "width": 750,
    }
  }
>
  <View
    accessibilityComponentType={undefined}
    accessibilityLabel={undefined}
    accessibilityTraits={undefined}
    accessible={true}
    collapsable={undefined}
    hitSlop={undefined}
    isTVSelectable={true}
    nativeID={undefined}
    onLayout={undefined}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Object {
        "opacity": 1,
      }
    }
    testID={undefined}
    tvParallaxProperties={undefined}
  >
    <View
      style={
        Object {
          "alignItems": "center",
          "backgroundColor": "black",
          "borderRadius": 50,
          "flex": 0,
          "height": 100,
          "justifyContent": "center",
          "width": 100,
        }
      }
    >
      <Text
        accessible={true}
        allowFontScaling={true}
        disabled={false}
        ellipsizeMode="tail"
        style={
          Object {
            "color": "white",
            "fontSize": 30,
          }
        }
      >
        1
      </Text>
    </View>
  </View>
</View>
`;
